### DISCUSSION TITLE
Labels for circuits

### DISCUSSION THREAD

**Discussion by FredrikM97** [FakeFishGames/Barotrauma#13781] [Ideas and suggestions]
Hi!

When I work with circuits I want to add a label or a box around components within a circuit box. It is difficult to tell the purpose of a memory or a group of circuits at sight. I also want to change the name of input/output to something else than just signal_out_1. It get confusing over time if i have 5 wifi input but no info which is what without documenting on a paper.

Is this any feature that could be added to the multiplayer gameplay?

**Comment by PlasmaUwU** [comment]
This has already been implemented on the unstable version, it will be in the next update
